src/App.js

```diff
@@ -1,25 +1,101 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import styled from 'styled-components'
+import Header from './Components/Header'
+import MedicationGroup from './Components/MedicationGroup'
+
+export default function App() {
+  const currentDate = new Date()
+
+  const medication = [
```

Comment:
How about creating a json file and import the array from there? That way it would be a bit cleaner. Also, I'd not put it in your function but outside the function.

Reply:
the array will be gone in the next user story